.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,28 @@
-<!-- ^^^^^
-Please provide a concise, informative and self-explanatory title.
-Don't put issue numbers in there, do this in the PR body below.
-For example, instead of "Fixes #1234" use "Introduce new method to calculate 1+1"
--->
-### 📚 Description
+<!-- Please provide a concise, informative and self-explanatory title. -->
+<!-- Don't put issue numbers in the title. Put it in the Description below. -->
```

Comment:
No strong preference, but maybe we should leave the example in here for the time being?

Reply:
Okay. Done.